components/UI/Drawer/Drawer.tsx

```diff
@@ -125,6 +125,22 @@ const Drawer = () => {
       geometry.location.lng,
     ]).format();
 
+    const openSourceUrl = () => {
+      if (source.channel === "twitter" && source.extra_parameters) {
+        const extraParams = JSON.parse(source.extra_parameters);
+        if (extraParams.user_id && extraParams.tweet_id) {
+          window.open(
+            `https://twitter.com/${extraParams.user_id}/status/${extraParams.tweet_id}`
+          );
+        }
+        return;
+      }
+      window.alert("kaynak açılamadı");
```

Comment:
maybe we can handle these issues by using an alert component. Did you try that before?